lib/data/analytics.js

```diff
@@ -30,7 +30,20 @@ const metricsTemplate = {
     "num_unique_viewers": {},
     "num_unique_collectors": {},
     "database_size": {},
-    "uses_external_db": 0
+    "uses_external_db": 0,
+    "sso_enabled": 0,
+    "num_client_audit_attachments": {
+      "recent": 0,
+      "total": 0
+    },
+    "num_client_audit_attachments_unprocessed": {
```

Comment:
I'm sort of reinterpreting all this as a "processing failed" scenario. Where attachment was attempted, but failed, so that is how they are "unprocessed".. but it shouldn't/doesn't count events that are unprocessed because they are new in the queue or something. Does that sound like what you intended, @matthew-white?

Reply:
Yes, that sounds like a useful way to think about it! The number of `submission.attachment.update` events that are related to a client audit log and that failed to be processed.

Reply:
It's a subset of the records counted in `failed5` I guess.